docs/source/faq.rst

```diff
@@ -714,3 +714,66 @@ Optuna may sometimes suggest parameters evaluated in the past and if you would l
 
     study = optuna.create_study()
     study.optimize(objective, n_trials=100)
+
+How can I sample the best expected parameters based on the evaluated trials?
+----------------------------------------------------------------------------
+
+As the optimization often overtunes the parameters of an objective function with an observational noise, it is convenient to sample the best parameters based on the debiased objective function.
+:class:`~optuna.samplers.GPSampler` enables us to sample the best expected parameters using the mean estimation of parameters.
+Note that the best expected parameters are defined as :math:`\mathrm{arg}\min_{x \in X} \mathbb{E}_{f}[f(x)]` where :math:`f` follows the Gaussian process.
```

Comment:
Is $X$ is a set of suggested params in a trial and $x$ is a one param in the suggested params as in the example below? Can we clarify $X$ and $x$ here?